### ISSUE TITLE
APK doesn't download in Default browser

### ISSUE THREAD

**Issue by Harishs192** [projectOpenRAP/OpenRAP#18]
1. Connected device to ekstepORAP network.
2. Gets a notification on sign on to ekstepORAP.
3. Redirects to APK home page.
4. APK doesn't gets downloaded in the default browser

**Comment by Harishs192** [comment]
Not Chrome, it was an  inbuilt android browser, 

As soon as you connect to ekstepORAP will get a notification from android o signup.

**Comment by siddharthmunot** [comment]
We will have two text set for captive portal browser and Normal Broswer..

**Comment by joel-ekstep** [comment]
@siddharthmunot We had a meeting yesterday with APSSDC - and we saw that even if we had used Chrome, and tried to open www.google.com, it showed the page-unavailable message. It was only when we tried to visit a page not seen before (www.xyz.com), that it showed the captive portal.

I think we need to see if there's a solution to this. It will be hard to assume that users will make the leap to:
1. Open chrome and visit a page. (There captive portal browser app download itself should work).
2. Open a website not visited before. (The captive portal should show up no matter what page is requested from normal browser).
